README.md

```diff
@@ -1,64 +1,5 @@
 # Tests for Mozilla's Treeherder
-This repository contains tests for [Mozilla's Treeherder](https://treeherder.mozilla.org/).
+This repository holds automation code that was used for Mozilla's Treeherder website. All code has been moved to the [primary Treeherder repo](https://github.com/mozilla/treeherder).
```

Comment:
I would argue that it still holds code in the commit history & also matches previous deactivated repos such as https://github.com/mozilla/oneanddone.  I can be swayed if you feel strongly about it.